### ISSUE TITLE
predict.mppm fails for marked patterns

### ISSUE THREAD

**Issue by marcelinodelacruz** [spatstat/spatstat#137]
`predict.mppm()` fails for models for replicated marked patterns. This happens either for  models including or not including marks as a predictor, e.g.:

```R
H<- hyperframe(lansingrep = list(lansing, lansing, lansing))
m <- mppm(lansingrep ~ marks + x, H)
predict(m)   
```

>Error: data points are marked, but dummy points are not

```R
m <- mppm(lansingrep ~ x, H)
predict(m)   
```

>Error: data points are marked, but dummy points are not

**Comment by baddstats** [comment]
The `mppm` code does not yet fully support marked point patterns. 
We will take this as a feature request.

**Comment by baddstats** [comment]
This should work now (in `spatstat 1.64-1.025`)

**Comment by baddstats** [comment]
This is now implemented (in the development version, `1.64-1.025` and later)
